CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We welcome and encourage contributions. We are open minded to suggestions, encourage debate, and aim to build a consensus; if there is no consensus, maintainers will have the final say.
```

Comment:
Worth saying who the 'we' is in this first usage?

Reply:
I'd have thought that was fairly clear. Not sure what else we'd put:
- BBC is too generic
- Honey Badger only makes sense internally
- Bigscreen Player maintainers is a bit wordy

Reply:
Yep, fair point 👍